src/Http/Middleware/RequestLoggingMiddleware.php

```
@@ -145,22 +168,85 @@ abstract protected function addServiceContext(Request $request, array $context):
     }
 
     /**
      * Log the completion of a request.
-     * Override this method to enable request completion logging.
+     * Provides default implementation - override if needed.
      *
      * @param  mixed  $response
+     * @param  array  $metrics Performance metrics
      */
-    protected function logRequestComplete(Request $request, $response, array $context): void
+    protected function logRequestComplete(Request $request, $response, array $context, array $metrics = []): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$context'. (undefined)

(UnusedFormalParameter)